lib/rules/presentation-role-conflict-matches.js

```diff
@@ -0,0 +1,7 @@
+import { getImplicitRole } from '../commons/aria';
+
+function presentationRoleConflictMatches(node) {
```

Comment:
Can you use the virtualNode here instead? We've been moving our rules over to exclusively use virtualNode.  Tests can run dramatically faster if the DOM isn't involved, and not all situations where axe-core is used needs to build a complete DOM tree.